src/main/java/com/powsybl/openloadflow/dc/equations/AbstractClosedBranchDcFlowEquationTerm.java

```diff
@@ -54,6 +64,18 @@ protected AbstractClosedBranchDcFlowEquationTerm(LfBranch branch, LfBus bus1, Lf
         }
     }
 
+    /**
```

Comment:
This part (aimed at updating the power according to the branch's current tap) should surely be implemented in a listener. As I'm not comfortable with OLF's listener management, the idea is to first validate the operation before implementing it more cleanly (if necessary). 

I note that this development should also help the “slow” DC SA, which doesn't seem to properly take into account the change in impedance, induced by modifying the tap via a PST action. So this could be dealt with in a separate PR.

Reply:
If there is not managed by the listerner mechanism, how does the lastTapPosition work with the DcIncrementalPhaseControlOuterLoop ?

Reply:
It seems to me that the right way to do this would be to catch the tapChange events and update the power in the equation term.

Reply:
And for the "optimized run" that doesn't use the notifications but clones the target vector (is it really faster by the way ?) then call explicitely the eqaution terms attached to the PST modified by the actions.

Reply:
After more thoughts I still don't know what is best between a notification based implem that would catch the tap change events, that more complex, and an "always recompute approach" that is simple and good enough if the equation is not called to frequently.

Reply:
I note that in the current state of the PR, the power is only updated when there is a call with a tap position different from the current tap of the equation term. If notification based implem is chosen, it would be better to catch the tapChange events to apply the modification, as you mentioned above.

Reply:
Closed by https://github.com/powsybl/powsybl-open-loadflow/pull/1095/commits/c03d5665c12b1526ebcd99b35cfde5acacb2a123